python/cuml/test/conftest.py

```diff
@@ -63,6 +65,33 @@ def housing_dataset():
     return X, y, feature_names
 
 
+@pytest.fixture(scope="session")
+def random_seed(request):
+    random_seed = np.random.randint(0, 1e6)
+    os.environ['PYTEST_RANDOM_SEED'] = str(random_seed)
```

Comment:
This sets the env var, but could it log the seed too? Maybe just a print() here? I see it shows up on error, but seems fairly harmless to log it all the time.

Also, could users override this by setting that environment variable? I think we want an ability to override as well right?

Reply:
Probably want to check whether PYTEST_RANDOM_SEED is set (and an int) and use it if possible otherwise do this and print it.